lib/run.tsx

```diff
@@ -86,17 +86,17 @@ const run = async (string: string, _opts: RunOpts = {}) => {
   }
 
   return {
-    default: () => (
+    default: props => (
       <Contexts terms={terms} baseUrl={baseUrl} variables={variables}>
         <Components.Style stylesheet={stylesheet} />
-        <Content />
+        <Content {...props} />
       </Contexts>
     ),
     toc,
-    Toc: () =>
+    Toc: props =>
       Toc ? (
         <Components.TableOfContents>
-          <Toc />
+          <Toc {...props} />
```

Comment:
Is this our `<Toc>` component? Why does _it_ need `props`?

Reply:
It doesn't really. But since it's technically override-able, I thought it might as well get them.

Reply:
Yeah that's what I figured. Makes sense!